app/containers/ItunesContainer/tests/reducer.test.js

```diff
@@ -44,4 +46,52 @@ describe('ItunesContainer reducer tests', () => {
       })
     ).toEqual(expectedResult);
   });
+  it('should ensure when CLEAR_ARTIST is dispatched , it clears the trackDetails', () => {
+    const expectedResult = { ...state };
+    expect(
+      itunesContainerReducer(state, {
+        type: itunesContainerTypes.CLEAR_ARTIST
+      })
+    ).toEqual(expectedResult);
+  });
+  it('should return state when an action of REQUEST_GET_TRACK_DETAILS is dispatched', () => {
```

Comment:
Good with the tests. Phrase it better. I believe we are not just returning the state, we are updating the state with trackId, right?